### ISSUE TITLE
Wifi/power lost -> reconnect does not work 

### ISSUE THREAD

**Issue by dexter2111** [Qrome/marquee-scroller#219]
When the power is lost and came back, the ESP8266 start before the wifi router, WiFi message is showing on display.
A have to reconnect the power supply on ESP8266(does not auto-connect), after the router start.

**Comment by Qrome** [comment]
Not sure what is different about your network, but this is not an issue.  Please check the lease or of there is another authentication required to retain IP address.   What is the IP lease length on the router?

**Comment by dexter2111** [comment]
Hi Qrome,
The address lease time is 120 min.
I have 5 devices with Esp 8266, on my network, only with marquee-scroller, a have this issue.(static IP)
2 are Sonoff , and 3 are  Blynk , all with DHCP.
Thank you.

**Comment by Leonessa02016** [comment]
I have exactly the same situation.In my understanding the problem is that the Wi-Fi signal from the router takes more time with respect the time that esp8266 is looking for the network 8few seconds).
So is it possible to increase the time that esp8266 is looking for the network (from few seconds to some minutes) ?
Thanks you

**Comment by brianmiller1956** [comment]
I also have this problem, we have frequent power cuts, my router does not produce Wi-Fi output until it has had power on for about three minutes, at this point the scroller has stopped looking for it's configured network.

**Comment by dexter2111** [comment]
I have set the address lease time to maxim  value on the  wifi router,but that don't fix this issue.
The esp8266 should try connect to wifi, 3-4 times, with a delay of 1-2min before the next attempt.

**Comment by dexter2111** [comment]
@Qrome 
There  is a solution for this issue?
Thank you.

**Comment by Qrome** [comment]
No, not offering a solution for this.  You will simply have to start the device after the router and network are up and running.
